### ISSUE TITLE
README

### ISSUE THREAD

**Issue by adonespitogo** [felixsinger/mkinitcpio-overlayfs#1]
Hi,

I'm looking for solutions to mount the root file system as read-only in archlinux. I'm building an application for a SBC, Expressobin in particular. I want to protect the rootfs from corruption at some degree. Can you provide a basic instructions how to install your script on archlinux?

Thanks,
Adones

**Comment by felixsinger** [comment]
Yes... Sorry for that. Because of lack of time I haven't done this yet.

I think I will start doing this in 2 hours or so.
